mantidproject/mantid
# SaveNXcanSAS: Convert edges to points on save

**Description of work.**

In order to adhere to the [NXcanSAS standard](https://manual.nexusformat.org/classes/applications/NXcanSAS.html) we convert a histogram transmission array into points before saving to the file.

LoadNXcanSAS has been adapted to support loading bin edge data that mantid might have previously saved.

**To test:**

See bug replication instructions in the issue.

Fixes #30537 

*This does not require release notes* because **fill in an explanation of why**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
